docs/src/main/asciidoc/security-built-in-authentication.adoc

```diff
@@ -7,17 +7,45 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 
 include::./attributes.adoc[]
 
-This document describes the Quarkus built-in authentication mechanisms for HTTP based FORM, BASIC and Mutual TLS authentication as well as the proactive authentication.
+The following section describes the Quarkus built-in authentication mechanisms for HTTP-based BASIC, FORM, and both Mutual TLS and proactive authentication.
 
 [[basic-auth]]
 == Basic Authentication
+ 
+HTTP Basic authentication (BA) uses fields in the HTTP header, and is the easiest to set up and least resource demanding technique for enforcing access controls to Web resources without requiring HTTP cookies, session identifiers, or login pages.
+
+In the context of an HTTP request, basic access authentication is a method for an HTTP user agent (such as a Web browser) to provide a user name and password when making a request. In basic HTTP authentication, a request contains a header field in the form of _Authorization: Basic_ _<credentials>_, where credentials are the Base64 encoding of a user ID and password joined by a colon. 
+
+====
```

Comment:
We never used this construct without an admonition type (e.g. `[NOTE]`) so we need to check how it renders on the website.

/cc @ebullient

Reply:
@gsmet Do you mean this one, `_Authorization: Basic_ _<credentials>_` ?
I'd just go for a simple highlighting like `Authorization: Basic <credentials>`

Reply:
@sberyozkin I think he was talking about the ==== box I used to highlight the example.
The underscores mean a plain italic.

Reply:
Yes, what @MichalMaler said.

Reply:
`Authorization: Basic <credentials>` should be inline code, rather than italics, I believe. 

Without the type (e.g. NOTE), it is a [simple admonition](https://docs.asciidoctor.org/asciidoc/latest/blocks/example-blocks/#delimited) that should render as a plain callout box, but we will have to see what the web CSS does with it.